Makefile

```diff
@@ -24,10 +24,18 @@ nvim-treesitter/parser/lua.so: nvim-treesitter $(NEOVIM)
 			   -c "TSInstallSync lua" \
 			   -c "q"
 
+nvim-treesitter/parser/rust.so: nvim-treesitter $(NEOVIM)
```

Comment:
My comment somehow got deleted in the mess, but do you know how to avoid the duplication here @lewis6991?

Reply:
Yeah, I can do that. No massive need to do it in this PR.